NetCord/Application.cs

```diff
@@ -65,13 +44,177 @@ public Application(JsonModels.JsonApplication jsonModel, RestClient client) : ba
             IntegrationTypesConfiguration = integrationTypesConfiguration.ToDictionary(i => i.Key, i => new ApplicationIntegrationTypeConfiguration(i.Value));
     }
 
+    /// <summary>
+    /// The ID of the application.
+    /// </summary>
+    public override ulong Id => _jsonModel.Id;
+
+    /// <summary>
+    /// The name of the application.
+    /// </summary>
+    public string Name => _jsonModel.Name;
+
+    /// <summary>
+    /// The icon hash of the application.
+    /// </summary>
+    public string? IconHash => _jsonModel.IconHash;
+
+    /// <summary>
+    /// The description of the application.
+    /// </summary>
+    public string Description => _jsonModel.Description;
+
+    /// <summary>
+    /// The application's RPC origin URL list, where <c>T</c> is of type <see cref="string"/>. <see langword="null"/> if RPC is disabled.
+    /// </summary>
+    public IReadOnlyList<string> RpcOrigins => _jsonModel.RpcOrigins;
+
+    /// <summary>
+    /// When <see langword="false"/>, only the application owner can add the application to guilds.
+    /// </summary>
+    public bool? BotPublic => _jsonModel.BotPublic;
+
+    /// <summary>
+    /// When <see langword="true"/>, the application's bot will only join upon completion of the full OAuth2 code grant flow.
+    /// </summary>
+    public bool? BotRequireCodeGrant => _jsonModel.BotRequireCodeGrant;
+
+    /// <summary>
+    /// The partial <see cref="User"/> object of the application's bot.
+    /// </summary>
+    public User? Bot { get; }
+
+    /// <summary>
+    /// The application's Terms of Service URL.
+    /// </summary>
+    public string? TermsOfServiceUrl => _jsonModel.TermsOfServiceUrl;
+
+    /// <summary>
+    /// The application's Privacy Policy URL.
+    /// </summary>
+    public string? PrivacyPolicyUrl => _jsonModel.PrivacyPolicyUrl;
+
+    /// <summary>
+    /// The partial <see cref="User"/> object of the application's owner.
+    /// </summary>
+    public User? Owner { get; }
+
+    /// <summary>
+    /// The hex-encoded verification key used for interactions and the GameSDK's GetTicket.
```

Comment:
Might be better to do
```suggestion
    /// The hex-encoded verification key used for HTTP interactions and the GameSDK's GetTicket.
```